### PR TITLE
A new script for applying a ml model on a batch of past incidents

### PR DESCRIPTION
**Pull request by lperryp**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A new script for applying a ml model on a batch of past incidents.
Some changes were needed in DbotPredictPhishingWords to support prediction of multiple incidents 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by erezh31 — APPROVED** [review]
Looks good
Please add simple UT for the script

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 1]
```diff
@@ -13,13 +13,17 @@ args:
...
+    same as in the incidents query page (valid strings exaple: "3 days ago", ""2019-01-01T00:00:00
+    +0200")'
   isArray: false
   name: fromDate
   required: false
   secret: false
 - default: false
-  description: 'The end date by which to filter incidents. Date format will be the same as in the incidents query page (valid strings exaple: "3 days ago", ""2019-01-01T00:00:00 +0200")'
+  description: 'The end date by which to filter incidents. Date format will be the
+    same as in the incidents query page (valid strings exaple: "3 days ago", ""2019-01-01T00:00:00
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:25]: exaple -> example

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 2]
```diff
@@ -13,13 +13,17 @@ args:
   required: false
   secret: false
 - default: false
-  description: 'The start date by which to filter incidents. Date format will be the same as in the incidents query page (valid strings exaple: "3 days ago", ""2019-01-01T00:00:00 +0200")'
+  description: 'The start date by which to filter incidents. Date format will be the
+    same as in the incidents query page (valid strings exaple: "3 days ago", ""2019-01-01T00:00:00
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:17]: exaple -> example

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 3]
```diff
@@ -54,21 +58,24 @@ args:
...
+  description: The field name with the label. Supports a comma-separated list, the
+    first non-empty value will be taken.
   isArray: false
   name: tagField
   required: true
   secret: false
 - default: false
   defaultValue: '15'
-  description: Sample text of which the total number words are less than or equal to this number will be ignored.
+  description: Sample text of which the total number words are less than or equal
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:69]: Sample text of which the total number of words...

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 4]
```diff
@@ -54,21 +58,24 @@ args:
...
+    to this number will be ignored.
   isArray: false
   name: removeShortTextThreshold
   required: false
   secret: false
 - default: false
   defaultValue: '0.99'
-  description: Remove emails with similarity greater then this threshold, range 0-1, where 1 is completly identical.
+  description: Remove emails with similarity greater then this threshold, range 0-1,
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:77]: Remove emails with similarity greater than...

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 5]
```diff
@@ -87,7 +94,15 @@ args:
   secret: false
 - default: false
   defaultValue: '*'
-  description: 'A comma-separated list of email tags values and mapping. The script considers only the tags specified in this field. You can map label to another value by using this format: LABEL:MAPPED_LABEL. For example, for 4 values in email tag: malicious, credentials harvesting, inner communitcation, external legit email, unclassified. While training, we want to ignore "unclassified" tag, and refer to "credentials harvesting" as "malicious" too. Also, we want to     merge "inner communitcation" and "external legit email" to one tag called "non-malicious". The input will be: malicious, credentials harvesting:malicious, inner communitcation:non-malicious, external legit email:non-malicious'
+  description: 'A comma-separated list of email tags values and mapping. The script
+    considers only the tags specified in this field. You can map label to another
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:98]: You can map a label

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 6]
```diff
@@ -87,7 +94,15 @@ args:
...
+    value by using this format: LABEL:MAPPED_LABEL. For example, for 4 values in email
+    tag: malicious, credentials harvesting, inner communitcation, external legit email,
+    unclassified. While training, we want to ignore "unclassified" tag, and refer
+    to "credentials harvesting" as "malicious" too. Also, we want to     merge "inner
```
Comment (kirbles19) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:102]: remove extra spaces

File: Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml [hunk 7]
```diff
@@ -0,0 +1,119 @@
+args:
+- default: false
+  description: Additional text by which to query incidents.
+  isArray: false
+  name: query
+  required: false
+  secret: false
+- default: false
+  defaultValue: Phishing
+  description: A common-separatetd list of incident types by which to filter.
```
Comment (kirbles19) [inline Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml:10]: A comma-separated list....

File: Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+- default: false
+  description: If non-empty, hash every word with this seed.
+  isArray: false
+  name: hashSeed
+  required: false
+  secret: false
+- default: false
+  defaultValue: '*'
+  description: 'A comma-separated list of email tags values and mapping. The script
+    considers only the tags specified in this field. You can map label to another
```
Comment (kirbles19) [inline Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml:54]: You can map a label....

File: Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+    value by using this format: LABEL:MAPPED_LABEL. For example, for 4 values in email
+    tag: malicious, credentials harvesting, inner communitcation, external legit email,
+    unclassified. While training, we want to ignore "unclassified" tag, and refer
+    to "credentials harvesting" as "malicious" too. Also, we want to     merge "inner
```
Comment (kirbles19) [inline Packs/ML/Scripts/DBotPredictIncidentsBatch/DBotPredictIncidentsBatch.yml:58]: Remove extra spaces

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@lperryp Done.

**Pull request merged**